maintainer/CI/build_cmake.sh

```diff
@@ -94,7 +94,7 @@ set_default_value check_skip_long false
 set_default_value make_check_tutorials false
 set_default_value make_check_samples false
 set_default_value make_check_benchmarks false
-set_default_value python_version 2
```

Comment:
does it make sense to have this variable at all after requiring version 3?

Reply:
At the moment we still allow compilation with Python2 bindings, we just offer no guarantee that it will work. Removing the ability to use Python2 means removing all `from __future__` imports, simplifying a few exceptions statements, removing `object` from classes, simplifying the CMake logic, removing the line you just flagged from GitLab CI and Travis CI files, etc.

Reply:
I think we can remove it from the CI scripts already. Users shouldn’t be running those anyway.

Reply:
I've removed the variable.